src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Octokit;
+
+namespace Microsoft.Docs.Build
+{
+    internal class GitHubAccessor
```

Comment:
>class GitHubAccessor [](start = 13, length = 20)

make it static?

Reply:
My later commits will pass token to this class, so leave it non static for now.

---
In reply to: [207127474](https://github.com/dotnet/docfx/pull/3144#discussion_r207127474) [](ancestors = 207127474)